### DISCUSSION TITLE
ETH bridge poblem

### DISCUSSION THREAD

**Discussion by abdullahansari-yk** [Cyfrin/foundry-full-course-cu#2489] [Q&A]
As patrick showed in the video , I have also bridge some ETH from my **sepolia tesnet** to **zksync sepolia testnet.**
But I haven't received anything still now.
Because of this, I am unable to deploy my contract on zksynck (there is nothing to work on).
So Community, What should I do?

**Comment by Bamservice47** [comment]
is it still the same ?
i need about 0.4 sETH
to bridge to zksync
0xb3f7c120245d92B9c8D3f0edbA59f5a89fc5debb

**Comment by EngrPips** [comment]
Get more sepolia ETH from chainlink faucet https://faucets.chain.link/sepolia and retry. also I bridged and didn't run into any problem whatsoever
